internal/sbi/producer/nf_management.go

```diff
@@ -246,21 +248,29 @@ func GetNFInstancesProcedure(nfType string, limit int) (*nrf_context.UriList, *m
 		}
 		return nil, problemDetail
 	}
-	logger.NfmLog.Infoln("UL: ", UL)
-	originalUL := &nrf_context.UriList{}
-	if err := mapstructure.Decode(UL, originalUL); err != nil {
-		logger.NfmLog.Errorf("Decode error in GetNFInstancesProcedure: %+v", err)
-		problemDetail := &models.ProblemDetails{
-			Title:  "System failure",
-			Status: http.StatusInternalServerError,
-			Detail: err.Error(),
-			Cause:  "SYSTEM_FAILURE",
+	logger.NfmLog.Infoln("ULs: ", ULs)
+	rspUriList := &nrf_context.UriList{}
+	for _, UL := range ULs {
+		originalUL := &nrf_context.UriList{}
+		if err := mapstructure.Decode(UL, originalUL); err != nil {
+			logger.NfmLog.Errorf("Decode error in GetNFInstancesProcedure: %+v", err)
+			problemDetail := &models.ProblemDetails{
+				Title:  "System failure",
+				Status: http.StatusInternalServerError,
+				Detail: err.Error(),
+				Cause:  "SYSTEM_FAILURE",
+			}
+			return nil, problemDetail
+		}
+		rspUriList.Link.Item = append(rspUriList.Link.Item, originalUL.Link.Item...)
+		if rspUriList.NfType == "" && originalUL.NfType != "" {
+			rspUriList.NfType = originalUL.NfType
 		}
-		return nil, problemDetail
 	}
-	nrf_context.NnrfUriListLimit(originalUL, limit)
+
+	nrf_context.NnrfUriListLimit(rspUriList, limit)
 	// c.JSON(http.StatusOK, originalUL)
```

Comment:
Nitpick: This code has been commented out and can be removed.

Reply:
@BENHSU0723 
Please help to remove it.